keel-sql/src/main/resources/db/changelog/20201006-add-unhealthy-repository.yml

```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-unhealthy-repo
+      author: emjburns
+      changes:
+        - createTable:
+            tableName: unhealthy
+            columns:
+              - column:
+                  name: resource_id
+                  type: varchar(255)
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: time_detected
+                  type: bigint
```

Comment:
We went through some trouble to update all timestamp columns to `TIMESTAMP` a while back (https://github.com/spinnaker/keel/pull/1224). Why are diverging from that now?

Reply:
because I hate timestamps and had a hard time getting them working, and I think they're dumb. But I made @robfletcher help me and I'm changing this. But my stubborn opinion remains :D